man/systemd.exec.xml

```diff
@@ -1242,28 +1242,66 @@ RestrictNamespaces=~cgroup net</programlisting>
         stopped. This setting is implied if <varname>DynamicUser=</varname> is set.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>PrivateMounts=</varname></term>
+
+        <listitem><para>Takes a boolean parameter. If set, the processes of this unit will be run in their own private
+        file system (mount) namespace with all mount propagation from the processes towards the host's main file system
+        namespace turned off. This means any file system mount points established or removed by the unit's processes
+        will be private to them and not be visible to the host. However, file system mount points established or
+        removed on the host will be propagated to the unit's processes. See <citerefentry
+        project='man-pages'><refentrytitle>mount_namespaces</refentrytitle><manvolnum>7</manvolnum></citerefentry> for
+        details on file system namespaces. When turned on, this executes three operations for each invoked process: a
```

Comment:
This text is so dense with information, that it's pretty hard to read, especially that this is a tough subject. I think it'd help if the text was split into paragraphs:
```

        <listitem><para>Takes a boolean parameter. If set, the processes of this unit will be run in their own private
        file system (mount) namespace with all mount propagation from the processes towards the host's main file system
        namespace turned off. This means any file system mount points established or removed by the unit's processes
        will be private to them and not be visible to the host. However, file system mount points established or
        removed on the host will be propagated to the unit's processes. See <citerefentry
        project='man-pages'><refentrytitle>mount_namespaces</refentrytitle><manvolnum>7</manvolnum></citerefentry> for
        details on file system namespaces. Defaults to off.  </para>

      <para>When turned on, this executes three operations for each invoked process: a
        new <constant>CLONE_NEWNS</constant> namespace is created, after which all existing mounts are remounted to
        <constant>MS_SLAVE</constant> to disable propagation from the unit's processes to the host (but leaving
        propagation in the opposite direction in effect). Finally, the mounts are remounted again to the propagation
        mode configured with <varname>MountFlags=</varname>, see below.</para>

        <para>File system namespaces are set up individually for each process
        forked off by the service manager. Mounts established in the namespace of the process created by
        <varname>ExecStartPre=</varname> will hence be cleaned up automatically as soon as that process exits and will
        not be available to subsequent processes forked off for <varname>ExecStart=</varname> (and similar applies to
        the various other commands configured for units). Similarly, <varname>JoinsNamespaceOf=</varname> does not
        permit sharing kernel mount namespaces between units, it only enables sharing of the <filename>/tmp</filename>
and <filename>/var/tmp</filename> directories. </para>

       <para>Other file system namespace unit
        settings — <varname>PrivateMounts=</varname>, <varname>PrivateTmp=</varname>,
        <varname>PrivateDevices=</varname>, <varname>ProtectSystem=</varname>, <varname>ProtectHome=</varname>,
        <varname>ReadOnlyPaths=</varname>, <varname>InaccessiblePaths=</varname>, <varname>ReadWritePaths=</varname>
        — implicitly enable this option. Hence it is primarily useful to explicitly request this behaviour if none of the other settings are used.</para>
```
(I think we need to keep the list of setting that imply PrivateMounts=true up-to-date here.)

Reply:
hmm, your reworked version changes that those other options turn on file system namespacing equivalent to this option, and replaces it with that they imply this option. But it really doesn't, as PrivateMounts= will show up off in the bus properties and elsewhere... We had bug reports about this in other context, so i figured I should be more correct here instead of sloppily claiming that the other options imply this one.

(And yes, previously we haven't been that accurate in cases like this, but I figure we should)

I'll use your version of the rest, and work in this more exact wording regarding the implying due to other options.